pkg/pillar/cmd/upgradeconverter/UUIDPairToNumConvert.go

```diff
@@ -42,17 +42,17 @@ func convertUUIDPairToNum(ctxPtr *ucContext) error {
 	}
 	defer pubUUIDPairToNum.Close()
 
-	pubUUIDPairAndIfIdxToNum, err := ctxPtr.ps.NewPublication(pubsub.PublicationOptions{
+	pubAppInterfaceToNum, err := ctxPtr.ps.NewPublication(pubsub.PublicationOptions{
```

Comment:
That's fine as the output type, but shouldn't the input type be UUIDPairAndIfIdxToNum? That is what we are currently using.

I don't know if we will need to also support the older systems where we used UuidPairToNum i.e., do a two-step upgrade. Makes sense to check when the UUIDPairAndIfIdxToNum was introduced.

Reply:
The underlying type is still `UUIDPairAndIfIdxToNum` - that is the name that pubsub will get when it will be constructing the destination directory. `AppInterfaceToNum` is just an alias for software developers to use, it has not effect on the compiled code.

Reply:
OK, but it would be good to add some comments about this to the upgradeconverter code because it is easy to get confused by the alias.